build/deploy/cockroach.sh

```diff
@@ -2,13 +2,13 @@
 
 set -eu
 
-if [ -f ./test.sh ]; then
-  ./test.sh
+if [ -f ../build/test.sh ]; then
+  ../build/test.sh
 fi
 
 if [ "${1:-}" = "shell" ]; then
   shift
-  /bin/bash "$@"
+  /bin/sh "$@"
```

Comment:
Why change this to `sh`? We use `bash` in all our `#!` lines and use bashisms, well, unabashedly.